components/dropdown/demo/overlay-visible.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 6
-title: 菜单隐藏方式
+title:
+  zh-CN: 菜单隐藏方式
+  en-US: The way of hiding menu.
 ---
 
+## zh-CN
+
 默认是点击关闭菜单，可以关闭此功能。
 
+## en-US
+
+The default is to close the menu when you click on menu items, this feature can be turned off.
```

Comment:
I am not sure, see:

![image](https://cloud.githubusercontent.com/assets/3580607/16445267/80e961b8-3e14-11e6-9081-1cbd6f8b3e08.png)